modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -24,6 +26,22 @@
 
 
 def apply_list_of_funcs(funcs, df):
+    """
+    Execute all operations stored in the call queue on the partition in a worker process.
+
+    Parameters
+    ----------
+    funcs : list
+        A call queue that needs to be executed on the partition.
+    df : pandas.DataFrame
+        A pandas DataFrame the call queue needs to be executed on.
+
+    Returns
+    -------
+    tuple
+        Tuple of result pandas DataFrame and node IP address
+        where ``apply_list_of_funcs`` was executed.
```

Comment:
I think there's specific numpydoc syntax to describe tuple returns in a piece-by-piece basis with types, etc.

Reply:
fixed